hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -468,8 +476,12 @@ private Partitioner getPartitioner(HoodieTable table, boolean isUpsert, Workload
     // cache writeStatusRDD before updating index, so that all actions before this are not triggered again for future
     // RDD actions that are performed after updating the index.
     writeStatusRDD = writeStatusRDD.persist(config.getWriteStatusStorageLevel());
+    indexLookupTimer = metrics.getIndexLookupCtx();
     // Update the index back
     JavaRDD<WriteStatus> statuses = index.updateLocation(writeStatusRDD, jsc, table);
+    metrics.updateIndexLookupMetrics(metrics.getDurationInMs(indexLookupTimer == null ? 0L :
```

Comment:
we should report this a separate indexUpdate metric, no?

Reply:
Sure when sending the metrics from metric.updateIndexMetrics("Index Lookup",time) and metrics.updateIndexMetrics("Index Update", time) would be the best way to handle this is what I think. If you agree I will make the changes right away.

Reply:
We can still reuse the Timer.Context object we don't need another one for indexUpdate

Reply:
I believe so. is it possible for you to test this out and emit metrics to a graphite backend.

Reply:
PR updated. Sorry PR got closed by mistake while writing a comment

Reply:
Hi @vinothchandar is this good to go?